File: dmr-api/Models/PeriodDispatch.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DMR_API.Models
{
    public class PeriodDispatch
    {
        public int ID { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
  
        public bool IsDelete { get; set; }
        public int CreatedBy { get; set; }
        public int DeletedBy { get; set; }
        public int UpdatedBy { get; set; }

        public DateTime CreatedTime { get; set; }
        public DateTime? DeletedTime { get; set; }
        public DateTime? UpdatedTime { get; set; }
        public int PeriodMixingID { get; set; }
        public virtual PeriodMixing PeriodMixing { get; set; }
    }
}

File: dmr-app/src/app/_core/_model/period.dispatch.ts
import { PeriodMixing } from "./period.mixing";

export interface PeriodDispatch {
    id: number;
    startTime: any;
    endTime: any;
    isDelete: boolean;
    createdBy: number;
    deletedBy: number;
    updatedBy: number;
    createdTime: Date;
    deletedTime: Date | null;
    updatedTime: Date | null;
    periodMixingID: number;
    periodMixing: PeriodMixing;
}
